runtime/api/android/pbandk-runtime.api

```diff
@@ -913,7 +913,6 @@ public final class pbandk/wkt/EnumOptions : pbandk/ExtendableMessage {
 	public final fun getDeprecated ()Ljava/lang/Boolean;
 	public fun getDescriptor ()Lpbandk/MessageDescriptor;
 	public fun getExtension (Lpbandk/FieldDescriptor;)Ljava/lang/Object;
-	public fun getExtensionFields ()Lpbandk/ExtensionFieldSet;
```

Comment:
These were improperly included in the public API because of limitations in earlier versions of the `binary-compatibility-validator` gradle plugin. The updated version of the plugin properly excludes these functions from the public API.